app/views/spree/admin/products/_tax_category_form.html.haml

```diff
@@ -1,5 +1,5 @@
 = f.field_container :tax_category_id do
   = f.label :tax_category_id, t(:tax_category)
   %br
-  = f.collection_select(:tax_category_id, Spree::TaxCategory.all, :id, :name, {:include_blank => Spree::Config.products_require_tax_category ? false : t(:none)}, {:class => "select2 fullwidth"})
+  = f.collection_select(:tax_category_id, Spree::TaxCategory.all, :id, :name, tax_category_dropdown_options(Spree::Config.products_require_tax_category), {:class => "select2 fullwidth"})
```

Comment:
`Spree::Config.products_require_tax_category` is being passed here as `require_tax_category` param of the method, however, we are not using the `required_tax_category` param in the method [here](https://github.com/openfoodfoundation/openfoodnetwork/pull/13009/files#diff-d138265f39484ab7f6c7947abeda57ffeec7d7c9e93be299e1a008dd77fa4d2bR2).